projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -246,10 +248,13 @@ spec:
             - name: {{ $key }}
               value: "{{ $value }}"
             {{- end }}
+
           volumeMounts:
             - name: secrets
               mountPath: "/etc/secrets"
               readOnly: true
+            - mountPath: "/datajobs_temp_storage"
```

Comment:
Maybe it's a good idea to expose mountPath as variable in values.yaml and then re-use it in both `envs:` and `volumeMounts:`.

Reply:
why would someone want to change the location? 
I don't want to add extra configuration unless there is a good use case for it.

Reply:
It's a bit safer in terms of future updates since it is not so obvious that the mountPath is passed as env var as well. But feel free to discard my comment.